src/platform/packages/shared/kbn-esql-validation-autocomplete/scripts/generate_function_definitions.ts

```diff
@@ -692,6 +692,29 @@ const enrichGrouping = (
   });
 };
 
+/** Enriches scalar functions that only accept a query parameter */
+const enrichQueryStringFunction = (functionDefinition: FunctionDefinition): FunctionDefinition => {
```

Comment:
@darnautov I think instead of creating a whole new enrich function we could use the existing `FunctionEnrichments` thing above

Reply:
changed to `functionEnrichments` in 6b0a6b08d7c79b2193a927e183bec226bbd50932